packages/core/src/browser/widgets/widget.ts

```diff
@@ -156,7 +156,7 @@ export function addEventListener<K extends keyof HTMLElementEventMap>(
 }
 
 export function addKeyListener<K extends keyof HTMLElementEventMap>(
-    element: HTMLElement,
+    element: HTMLElement | Window,
```

Comment:
it looks like `HTMLElementEventMap` has some events which are not present in `WindowEventMap`.

Reply:
OK, thanks for checking!

Reply:
why closing this?